examples/react-native/src/components/Stats.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { Button, View, Platform, Dimensions } from 'react-native';
-import { connectStats } from 'react-instantsearch/connectors';
+import { connectStats } from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.